src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -25,12 +35,17 @@ const StyledChip = styled(Chip)({
   },
 });
 
-export const ChannelClaimTableRow = ({ claimedChannel, channelType }: ChannelClaimTableRowProps) => {
+export const ChannelClaimTableRow = ({ claimedChannel, channelType, isOnSettingsPage }: ChannelClaimTableRowProps) => {
   const { t } = useTranslation();
+  const user = useSelector((store: RootState) => store.user);
+  const customerId = user?.customerId ?? '';
+  const customerIdentfier = customerId ? getIdentifierFromId(customerId) : '';
```

Comment:
typo, pluss overflødig sjekk som håndteres i funksjonen
```suggestion
  const customerIdentifier = getIdentifierFromId(customerId);
```